packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -22,7 +22,7 @@
       :min-panel2-width="minWidths.reporter"
       :min-panel3-width="minWidths.aut"
       :show-panel1="runnerUiStore.isSpecsListOpen && !screenshotStore.isScreenshotting"
-      :show-panel2="!screenshotStore.isScreenshotting"
+      :show-panel2="!screenshotStore.isScreenshotting && !hideCommandLog"
```

Comment:
Just wondering, how come we're not mirroring the env var name? I.e. why `NO_COMMAND_LOG - hideCommandLog` instead of `HIDE_COMMAND_LOG - hideCommandLog` or `NO_COMMAND_LOG - noCommandLog` ?

Reply:
It started out that way, changed it based on @emilyrohrbough's [feedback here](https://github.com/cypress-io/cypress/pull/22641#discussion_r911969672) since in general boolean names are formed with verbs like `isThing` or whatever.

Reply:
Yeah I think this is marginal either way. Since tracing this takes you to the store, and the store shows how the value is initialized, I'm still just slightly in favor of "readability when somebody has to read/modify the Vue files" since that's probably the much more common situation for this code to be read.